### PR TITLE
Updated go-sdk to version 0.11.0

### PR DESCRIPTION
**Pull request by jackdelahunt**

### CODE REVIEW

**Comment by jackdelahunt** [comment]
Had to remove the multi-az option as there was no field for it anymore in the kafka request payload struct.

**Comment by jackdelahunt** [comment]
Cannot fix the GH action failing as you need to run `make docs` but you cannot use this tfplugindocs with an M1 mac.

https://github.com/hashicorp/terraform-plugin-docs/issues/41

File: Makefile [hunk 1]
```diff
@@ -4,7 +4,7 @@ NAMESPACE=pmuir
 NAME=rhoas
 BINARY=terraform-provider-rhoas
 VERSION ?= 0.1
-OS_ARCH ?= darwin_amd64
+OS_ARCH ?= darwin_arm64
```
Comment (wtrocki) [inline Makefile]: Not sure if we have this right

File: README.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # RHOAS Terraform Provider
 
+NOTE: Repository have been moved to https://github.com/redhat-developer/app-services-teraform-provider
```
Comment (wtrocki) [inline README.md:3]: ??

**Comment by wtrocki** [comment]
> Cannot fix the GH action failing as you need to run make docs but you cannot use this tfplugindocs with an M1 mac.

https://hub.docker.com/r/hashicorp/terraform/

File: Makefile (hunk 1 above)
Comment (jackdelahunt) [reply Makefile]: only way to get it to run on my machine, ideally we would have a check to figure out what os and architecture you are on

Comment (wtrocki) [reply Makefile]: https://stackoverflow.com/questions/394230/how-to-detect-the-os-from-a-bash-script

Comment (wtrocki) [reply Makefile]: Let's make things right. Changing this value to other value will invalidate it for others etc.

**Pull request merged**